From de7b2cb2b72f98329b231ceab2e98f21e8ddaa22 Mon Sep 17 00:00:00 2001
From: Pavel Raiskup <praiskup@redhat.com>
Date: Thu, 24 Sep 2015 11:03:45 +0200
Subject: [PATCH] bootstrap: fix race in temporary Makefile

Target 'bootstrap-deps' sometimes rebuilt $(bootstrap_files)
a bit earlier than the cleanup target 'bootstrap-deps-prep' was
invoked.  As a result, some of $(bootstrap_files) were missing.

* Makefile.am (bootstrap-deps): Rebuild $(bootstrap_files) after
bootstrap-deps-prep dependency.
---
 Makefile.am | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/Makefile.am b/Makefile.am
index 13dfc63a..e4d6074d 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -207,7 +207,9 @@ bootstrap_files = \
     $(ltversion_m4)
 
 .PHONY: bootstrap-deps bootstrap-deps-prep
-bootstrap-deps: bootstrap-deps-prep $(bootstrap_files)
+bootstrap-deps: bootstrap-deps-prep
+	$(MAKE) $(bootstrap_files)
+
 bootstrap-deps-prep:
 ## The following variables are substituted by 'bootstrap-dep-preps'
 	@exit_cmd=:; \
